container/state.go

```diff
@@ -68,6 +69,10 @@ func NewState() *State {
 
 // String returns a human-readable description of the state
 func (s *State) String() string {
+	if s.CommitInProgress > 0 {
+		return "Commit In Progress"
+	}
+
```

Comment:
I think this should not be here, because `State.String()` should return a state of the container life cycle.  `Commit In Progress` should not belong to a container's life cycle, it can happens on container `running`, `exited` and `paused`. 

And `return "Commit In Progress"` should be return `fmt.Sprintf("Commit In Progress")`